domain-cc/cc-app/src/python_src/api.py

```diff
@@ -111,3 +111,17 @@ def get_classification(claim: Claim) -> Optional[PredictedClassification]:
 
     log_as_json({"classification": classification})
     return classification
+
+
+@app.post("/claim-linker")
+def link_vbms_claim_id(claim_link_info: ClaimLinkInfo):
+    log_as_json(
+        {
+            "message": "linking claims",
+            "va_gov_claim_id": claim_link_info.va_gov_claim_id,
```

Comment:
i think if you wrap these in the with the `sanitize()` function it will clear that codeQL error